components/eventing-controller/controllers/subscription/beb/reconciler.go

```diff
@@ -9,7 +9,9 @@ import (
 	"strings"
 	"time"
 
+	"github.com/kyma-project/kyma/components/eventing-controller/pkg/handlers/beb"
 	"github.com/kyma-project/kyma/components/eventing-controller/pkg/handlers/sink"
+	utils2 "github.com/kyma-project/kyma/components/eventing-controller/pkg/handlers/utils"
```

Comment:
(nit) can we use better name than `utils2`?

Reply:
done